src/readme-file.ts

```diff
@@ -46,8 +49,8 @@ export class ReadmeFile {
           return {
             method,
             interface: int,
-            hookName: this.nameFactory.buildHookName(method),
-            suspenseHookName: this.nameFactory.buildHookName(method, {
+            hookName: buildHookName(method, this.service),
```

Comment:
This ended up sort of muddled within the changes, but early on I took out the `nameFactory` and replaced it with regular functions, mostly on the fact that not every method in the NameFactory actually needs both the service and options (clearing out some passthrough parameters for parts of the code that call these methods)

I can change this back though, I'm pretty ambivalent to either way and don't necessarily want to introduce change for the sake of change.